frontend/src/routes/ProtectedRoute.tsx

```diff
@@ -21,30 +22,20 @@ const ProtectedRoute: FC<Props> = ({ component: Component }) => {
   }
 };
 
-export const AuthorizedRoute = ({
-  currentUserRole,
-  allowedRoles,
-  isPMOSysAdmin = false,
-}: {
-  currentUserRole: string;
-  allowedRoles: string[];
-  isPMOSysAdmin?: boolean;
-}) => {
-  const { initialized, keycloak } = useKeycloak();
-  const location = useLocation();
-
-  if (initialized) {
-    return keycloak?.authenticated && currentUserRole ? (
-      allowedRoles.includes(currentUserRole) || isPMOSysAdmin ? (
+export const AuthorizedRoute = () => {
+  const { keycloak } = useKeycloak();
+  if (keycloak.authenticated) {
+    if (!keycloak?.tokenParsed?.client_roles) {
+      return <Unauthorized />;
+    } else {
+      return keycloak?.tokenParsed?.client_roles.includes("pmo-sys-admin") ? (
```

Comment:
So what if user doesn't have this role, doesn't seem like this includes any other role.